src/Uno.UI/UI/Xaml/Application.cs

```diff
@@ -65,6 +65,8 @@ public static class TraceProvider
 
 		public DebugSettings DebugSettings { get; } = new DebugSettings();
 
+		public ApplicationRequiresPointerMode RequiresPointerMode { get; set; } = ApplicationRequiresPointerMode.Auto;

```

Comment:
It still is not implemented, we probably should raise a message when setting the property.

Reply:
Setting the property has effect only on Xbox, it is ignored on other Windows SKUs (https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.application.requirespointermode?view=winrt-19041 in remarks). So we can just have it there and the fact that it does nothing is "expected".

Reply:
Ah well, if the docs say so ;)

Reply:
I have used the property before on desktop, so can confirm it does nothing :-D .